doravota/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "doravota",
+  "chain_id": "vota-ash",
+  "pretty_name": "Dora Vota",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "dora",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
already added. https://github.com/satoshilabs/slips/blob/master/slip-0173.md?plain=1#L87